### PR TITLE
V 1.27

### PR DESCRIPTION
**Pull request by neastin**

### CODE REVIEW

File: pvHelpers/email_util.py [hunk 1]
```diff
@@ -944,3 +945,11 @@ def setMIMEBcc(message, bccs):
         message.headers["Bcc"] = u"{}".format(", ".join([u"{} <{}>".format(bcc["display_name"], bcc["user_id"]) for bcc in bccs]))
 
     return True, mime.from_string(message.to_string())
+
+
+###FIXME: this works when we only have aliases for the 4 default maiboxes,
+# and use the real "DB" mailbox_name for a custom mailbox
+def setMailboxAlias(mailbox_name):
```
Comment (neastin) [inline pvHelpers/email_util.py]: call this `getMailboxAlias`

File: pvHelpers/email_util.py [hunk 2]
```diff
@@ -944,3 +945,11 @@ def setMIMEBcc(message, bccs):
...
+    if (mailbox_name in MAILBOX_ALIAS):
```
Comment (neastin) [inline pvHelpers/email_util.py]: don't need `()`. e.g  `if mailbox_name in MAILBOX_ALIAS:`

File: pvHelpers/email_util.py (hunk 1 above)
Comment (hu13) [inline pvHelpers/email_util.py]: done

File: pvHelpers/email_util.py (hunk 2 above)
Comment (hu13) [inline pvHelpers/email_util.py]: done

**Pull request merged**